__tests__/app.test.js

```diff
@@ -59,4 +60,49 @@ describe('app()', () => {
         });
     });
   });
+  describe('/api/articles/:article_id', () => {
+    it('200: return with a status of 200', () => {
+      return request(app).get('/api/articles/1').expect(200);
+    });
+    it('200: return with a status of 200 and the corresponding article 1 data', () => {
+      return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toHaveProperty('author', expect.any(String));
+          expect(article).toHaveProperty('title', expect.any(String));
+          expect(article).toHaveProperty('article_id', expect.any(Number));
+          expect(article).toHaveProperty('body', expect.any(String));
+          expect(article).toHaveProperty('topic', expect.any(String));
+          expect(article).toHaveProperty('created_at', expect.any(String));
+          expect(article).toHaveProperty('votes', expect.any(Number));
+          expect(article).toHaveProperty('article_img_url', expect.any(String));
+        });
+    });
+    it('200: return with a status of 200 and the corresponding article 5 data', () => {
+      return request(app)
+        .get('/api/articles/8')
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toHaveProperty('author', expect.any(String));
+          expect(article).toHaveProperty('title', expect.any(String));
+          expect(article).toHaveProperty('article_id', expect.any(Number));
+          expect(article).toHaveProperty('body', expect.any(String));
+          expect(article).toHaveProperty('topic', expect.any(String));
+          expect(article).toHaveProperty('created_at', expect.any(String));
+          expect(article).toHaveProperty('votes', expect.any(Number));
+          expect(article).toHaveProperty('article_img_url', expect.any(String));
+        });
+    });
+    it('400: return with a status of 400 when using the incorrect end point', () => {
+      return request(app)
+        .get('/api/articles/hello')
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe('Bad request');
+        });
+    });
+  });
```

Comment:
Nice error handling here for 400, don't forget to also handle the 404 error for api/articles/99999